conference.js

```diff
@@ -1459,6 +1464,22 @@ export default {
                     return;
                 }
 
+                // Add the camera track as a secondary track if a screenshare track was already added.
```

Comment:
is it possible to make this flow not depend on the desktop track? is the lack of old track not enough to tell that it needs to call add vs replace?

Reply:
No, lack of old track is not enough to tell if the code needs to call add or replace. Only if there is a desktop track, we would want to add the  camera as a new secondary track. Otherwise it will call the replaceTrack and use the first transceiver irrespective of whether is an existing camera track or not.